src/libs/tmdk/src/commands/deploy.ts

```diff
@@ -57,33 +58,31 @@ export const builder: CommandBuilder<Options, Options> = (yargs) =>
   });
 
 function syncReadFile(path: string) {
-  const result = fs.readFileSync(path, "utf-8");
-  return `${result}`; // TODO more idiomatic
+  return fs.readFileSync(path, "utf-8");
 }
```

Comment:
If this function is just a wrapper for readFileSync, you can declare the encoding as a constant and just use that function directly

Reply:
Agree, will update